src/shared/components/CSVExportButton.tsx

```diff
@@ -52,8 +52,9 @@ class CSVExportButton extends PureComponent<StateProps, {}> {
 
   private handleClick = () => {
     const {files} = this.props
+    const formatter = createDateTimeFormatter('YYYY-MM-DD-HH-mm')
```

Comment:
```suggestion
    const formatter = createDateTimeFormatter('YYYY-MM-DD HH:mm')
```

couple of changes to make this a proper format

Reply:
Just made the change! This makes more sense!